include/envoy/upstream/outlier_detection.h

```diff
@@ -81,8 +93,14 @@ class DetectorHostMonitor {
    * @return the success rate of the host in the last calculated interval, in the range 0-100.
    *         -1 means that the host did not have enough request volume to calculate success rate
    *         or the cluster did not have enough hosts to run through success rate outlier ejection.
+   * @param type specifies for which Success Rate Monitor the success rate value should be returned.
+   If the outlier detector is configured not to split external and local origin errors,
```

Comment:
nit: missing '*' at beginning of block comment lines.

Reply:
Done.